templates/email/brent/waste/container_request.html

```diff
@@ -0,0 +1,40 @@
+[%
```

Comment:
I think this file can be removed also?
Perhaps the message in Brent.pm about 'similar request was recently made' then needs updating as that might not always be the case.

Reply:
e55e9ba58ab677ba1433c00b57586b1d48dc3bec